src/debug/Debug.js

```diff
@@ -0,0 +1,112 @@
+import dat from 'dat.gui/build/dat.gui';
```

Comment:
Oh, OK, so more accurately, dat.gui's `package.json` is plain broken, referencing in its `main` a file using ES6 imports and needing Babel.